core/src/main/java/com/linecorp/armeria/server/RoutingTrie.java

```diff
@@ -564,4 +502,167 @@ static void validatePath(@Nullable String path) {
                           Integer.toHexString(KEY_CATCH_ALL), path);
         }
     }
+
+    /**
+     * A strategy to find values in {@link RoutingTrie}.
+     */
+    interface FindStrategy {
```

Comment:
Could be `private`?

Reply:
Currently, `FindStrategy` is used by `Routers.TrieRouter`.
https://github.com/line/armeria/blob/8b161ce2393dbe17f61b6c26569d64a4513a13aa/core/src/main/java/com/linecorp/armeria/server/Routers.java#L336-L339

Reply:
I already thought about `@FuntionalInterface` but I have no strong conviction that it is good for `@FuntionalInterface`. Because I think we could add other `find` method on this `interface`. WDYT?

Reply:
FindStrategy is gone. ⚰️